src/lib/redux-store/explore/explore-slice.ts

```diff
@@ -12,8 +13,10 @@ export interface LeaderboardValues {
 
 export type ActiveValues = Record<string, Array<[unknown, boolean]>>;
 
-export interface MetricsLeaderboardEntity {
+export interface MetricsExploreEntity {
   id: string;
+  measureIds: Array<string>;
+  // this is used to show leaderboard values
```

Comment:
I think this should be changed to `leaderboardMeasureId` to help future code readers